src/main/java/net/pms/io/OutputParams.java

```diff
@@ -28,103 +27,321 @@
+	private boolean shift_scr;
+	private boolean cleanup;
 
 	public OutputParams(PmsConfiguration configuration) {
 		if (configuration != null) {
-			waitbeforestart = configuration.getVideoTranscodeStartDelay() * 1000;
+			setWaitbeforestart(configuration.getVideoTranscodeStartDelay() * 1000);
 		} else {
-			waitbeforestart = 6000;
+			setWaitbeforestart(6000);
 		}
-		fromFrame = -1;
-		toFrame = -1;
-		secondread_minsize = 1000000;
+		setFromFrame(-1);
+		setToFrame(-1);
+		setSecondread_minsize(1000000);
 		if (configuration != null) {
-			minFileSize = configuration.getMinStreamBuffer();
-			minBufferSize = configuration.getMinMemoryBufferSize();
-			maxBufferSize = configuration.getMaxMemoryBufferSize();
+			setMinFileSize(configuration.getMinStreamBuffer());
+			setMinBufferSize(configuration.getMinMemoryBufferSize());
+			setMaxBufferSize(configuration.getMaxMemoryBufferSize());
 		}
-		if (maxBufferSize < 100) {
-			maxBufferSize = 100;
+		if (getMaxBufferSize() < 100) {
+			setMaxBufferSize(100);
 		}
-		timeseek = 0;
-		env = null;
+		setTimeseek(0);
+		setEnv(null);
 	}
 
 	/**
 	 * Set some values to allow fast streaming start of transcoded videos
 	 */
 	public void manageFastStart() {
-		if (mediaRenderer != null && mediaRenderer.isTranscodeFastStart()) {
-			waitbeforestart = 0; // no delay when the transcode is starting
-			minBufferSize = 1; // 1Mb of minimum buffer before sending the file
+		if (getMediaRenderer() != null && getMediaRenderer().isTranscodeFastStart()) {
+			setWaitbeforestart(0); // no delay when the transcode is starting
+			setMinBufferSize(1); // 1Mb of minimum buffer before sending the
+								 // file
 		}
 	}
 
 	@Override
 	public String toString() {
-		return "OutputParams [aid=" + aid +
-			", avidemux=" + avidemux +
-			", cleanup=" + cleanup +
-			", forceFps=" + forceFps +
-			", forceType=" + forceType +
-			", fromFrame=" + fromFrame +
-			", header=" + Arrays.toString(header) +
-			", hidebuffer=" + hidebuffer +
-			", input_pipes=" + Arrays.toString(input_pipes) +
-			", log=" + log +
-			", losslessaudio=" + losslessaudio +
-			", lossyaudio=" + lossyaudio +
-			", maxBufferSize=" + maxBufferSize +
-			", mediaRenderer=" + mediaRenderer +
-			", minBufferSize=" + minBufferSize +
-			", minFileSize=" + minFileSize +
-			", no_videoencode=" + no_videoencode +
-			", outputByteArrayStreamBufferSize= " + outputByteArrayStreamBufferSize +
-			", noexitcheck=" + noexitcheck +
-			", output_pipes=" + Arrays.toString(output_pipes) +
-			", secondread_minsize=" + secondread_minsize +
-			", shift_scr=" + shift_scr +
-			", sid=" + sid +
-			", stdin=" + stdin +
-			", timeend=" + timeend +
-			", timeseek=" + timeseek +
-			", toFrame=" + toFrame +
-			", waitbeforestart=" + waitbeforestart +
-			", workDir=" + workDir +
-			", env=" + env + "]";
+		return "OutputParams [aid=" + getAid() + ", avidemux=" + isAvidemux() + ", cleanup=" + isCleanup() + ", forceFps=" + getForceFps()
```

Comment:
Seemed better before? More readable

Reply:
@SubJunk automatic formatting. I will revert it.